.github/workflows/main.yml

```diff
@@ -1,5 +1,5 @@
 on:
-  push:    
+  push:
```

Comment:
@jurekseverin-isg 

can you please undo this change.

Reply:
Is the reason to have less "changed files" or why should it be removed? I would say removing trailing spaces is good though

Reply:
Yes that's the reason. I want to see only code changes and not the beautifying changes. But i think we should talk about a code style we want to use and then use a linter to ensure this code style. Or maybe you already a linter?

Reply:
I removed this change. All I use by now is ES-Lint, but it doesn't complain about trailing spaces, maybe it is configurable.